views/openacademy_menu.xml

```diff
@@ -27,4 +33,10 @@
         name="Sessions"
         parent="menu_root"
         action="action_window_session"/>
+
+    <menuitem id="menu_partner"
+        name="Partner"
```

Comment:
What is this for?

Reply:
This is because the partner view has been modified by inheritance. Please see **Exercise 12: Alter existing content**. For this reason, I needed to include a menu to display the partner form view.

Reply:
When you inherit a view, changes are applied to actions where the inherited view is used.

I think you don't need a new menu to see partners. Besides, an action that only displays a form view doesn't make much of sense, because a form view without an explicit ID won't display anything, just will prompt you to create a new record.

Please test changes functionally before requesting review, so you may detect this kind of details yourself before being reviewed.